apps/website/.vuepress/theme/global-components/ClrColorList.vue

```diff
@@ -20,7 +20,8 @@
           <div class="weight">{{ color.weight }}</div>
           <div class="color-code-container">
             <div class="color-code">
-              <cds-icon class="copy-icon" shape="copy"></cds-icon> {{ getColorInCode(color) }}
+              <cds-icon class="copy-icon" shape="copy" v-bind:style="{ fill: getForeground(color) }"></cds-icon>
```

Comment:
I think this is the wrong way to force an icon color, shouldn't we be using `--color` instead? https://clarity.design/storybook/core/?path=/story/components-icon-getting-started--page#custom-styles